pokeclicker/pokeclicker
# Items icons and search by item name in Held Item modal

## Description


Adds icons to the items in the Pokémon list in the held Item modal and the ability to search Pokémon who have certain items equipped.
Also increases the modal width.


## Motivation and Context



Visually ease the search of Pokémon with certain items equipped (i.e.: finding Pokémon with Power Bracer to change said item for a damage boosting item)


## How Has This Been Tested?



* Check in the modal if the items icons show up
* Use the search bar and change the search dropdown to see if searching by item works and searching by Pokémon didn't break.


## Screenshots (optional):

![image](https://github.com/pokeclicker/pokeclicker/assets/104547700/3cc62574-2afe-461f-ab34-b8d9b32d01a2)
![image](https://github.com/pokeclicker/pokeclicker/assets/104547700/32fd7341-f5fa-4b18-8943-d9e0e101977e)
![image](https://github.com/pokeclicker/pokeclicker/assets/104547700/fb7805fb-e3d4-463e-8a60-1c492aaeb266)



## Types of changes

- UI improvement
